src/nanoemoji/config.py

```diff
@@ -23,14 +23,47 @@
 import toml
 from typing import Any, Iterable, MutableMapping, NamedTuple, Optional, Tuple, Sequence
 
-
-FLAGS = flags.FLAGS
+from nanoemoji import write_font
```

Comment:
This seems like an inappropriate import to me, I think we should avoid it and that will conveniently avoid the problem with cycles

Reply:
removed the cycle